_haikus/Note_to_self.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: haiku
+title: Note to self
+author: Abubakar
+---
+
+
+Listen, be yourself.
```

Comment:
Please add a `<br>` tag or two spaces `  ` at the end of this line and the second line to force line breaks.

Reply:
I will do that.

Reply:
I have made the requested changes
Thank you for patiently reviewing my Haiku.